src/api/whoami.rs

```diff
@@ -50,7 +50,7 @@ pub async fn whoami(
                 Ok(found) => {
```

Comment:
For clarity, maybe it would make sense to rename this variable to `user`?

```suggestion
                Ok(user) => {
```